addon/mixins/in-viewport.js

```diff
@@ -215,19 +218,23 @@ export default Mixin.create({
   _bindScrollDirectionListener(sensitivity = 1) {
     assert('sensitivity cannot be 0', sensitivity);
 
-    const $contextEl = get(this, 'scrollableArea') ? $(get(this, 'scrollableArea')) : $(window);
+    const contextEl = get(this, 'scrollableArea') || window;
+    let elem = findElem(contextEl);
 
-    $contextEl.on(`scroll.directional#${get(this, 'elementId')}`, () => {
```

Comment:
Also concerned about the use of `elementId` here.  Perhaps this was there to solve a use case where this mixin was used on many components on a page.  So potentially, with this PR, there are multiple listeners on the window object.